src/lang/es.json

```diff
@@ -17,6 +17,12 @@
       "value": "symbol"
     }
   ],
+  "+CHvZG": [
```

Comment:
I'll assume this file is auto-generated.

Reply:
yup